tools/migration/import

```diff
@@ -42,7 +42,9 @@ class Admiral:
         self.token = token
 
     def __import_project(self, project, retry=True):
-        project_data = json.dumps({ "name": project.project_name, "isPublic": project.public })
+        project_data = json.dumps({ "name": project.project_name, "isPublic": project.public,
+                                    "customProperties": {"__enableContentTrust": False, "__preventVulnerableImagesFromRunning":True,
```

Comment:
I'm sorry, we should set the attribute "__preventVulnerableImagesFromRunning“ to False.